app/controllers/rails_webhooks/subscribers_controller.rb

```diff
@@ -54,7 +55,14 @@ def set_subscriber
 
       # Only allow a list of trusted parameters through.
       def subscriber_params
-        params.require(:subscriber).permit(:url, :active, :subscriptions)
+        params.require(:subscriber).permit(:url, :active, subscriptions: [])
       end
+
+    def set_subscriptions
+      params[:subscriber][:subscriptions] = params[:subscriptions].select { |_event, value| value == 'true' }.keys
```

Comment:
Before this we should check for the presence of `params[:subscriber][:subscriptions]`. If they are missing, return a validation error to the user.